src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.CodeGeneration.targets

```diff
@@ -118,6 +118,7 @@
       DebugTool="$(_RazorDebugGenerateCodeTool)"
       ToolAssembly="$(_RazorToolAssembly)"
       UseServer="$(UseRazorBuildServer)"
+      DisableFallback="$(_RazorDisableBuildServerFallback)"
```

Comment:
Call this `ForceServer`

Reply:
Or consider making UseServer a string

Reply:
Btw I thought about making `UseServer` a string but we are doing this only for tests and `UseServer` is a "public" switch and we don't want to make it too complicated to use. I am going with `ForceServer` approach